client/post-editor/style.scss

```diff
@@ -9,6 +9,18 @@
 	@include breakpoint( "<660px" ) {
 		padding: 0;
 	}
+
+	> .notice {
```

Comment:
Can we apply this to notice-list instead? Then you shouldn't need the direct descendant selector.